bin/crisper

```diff
@@ -12,27 +12,68 @@
 // jshint node: true
 'use strict';
 
-var nopt = require('nopt');
 var fs = require('fs');
 var path = require('path');
 var crisp = require('../index');
 
-var args = nopt(
+var cliArgs = require('command-line-args');
+
+var cli = cliArgs([
+  {
+    name: 'source',
+    alias: 's',
+    type: String,
+    defaultOption: true,
+    description: 'Input HTML file to split into HTML and JS files'
+  },
+  {
+    name: 'html',
+    alias: 'h',
```

Comment:
In 31f4509, I'm making sure any missing required flags will trigger the usage prompt.